extension/deployment/src/test/java/io/quarkiverse/temporal/deployment/WorkerNamedDefaultConfigTest.java

```diff
@@ -34,5 +42,32 @@ public void testNamedConfiguration() throws IllegalAccessException {
         // queue name default to worker name
         Worker worker = factory.getWorker("namedWorker");
         Assertions.assertNotNull(worker);
+        // worker config is not visible;
+        WorkerOptions options = (WorkerOptions) FieldUtils.readField(worker, "options", true);
+        Assertions.assertEquals("a0c88522ca7aa74f5e3a64e6adcef35c27af16ab", options.getBuildId());
```

Comment:
won't this change every time so we will have a flickering test failure?  Maybe better to assert its length of an expected git commit hash than the hash itself?

Reply:
this commit id has been mocked